app/src/controller/CategoriesController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Src\controller;
+
+use Src\router\Request;
+use Src\router\Response;
+use Src\service\CategoriesService;
+
+class CategoriesController
+{
+
+    /**
+     * Gets all news from the database
+     *
+     * @param Request $req
+     * @param Response $res
+     * @return void
+     */
+    public static function getCategories(Request $req, Response $res): void
+    {
+        $user = $req->getUser();
+        $category = new CategoriesService();
+        $data = $category->getAllCategories();
+
+        $res->toJSON([
+            'data' => $data,
+        ]);
+    }
+
+    /**
+     * TO-DO
+     * - reimplement responses with code and sutff
+     */
+    public static function addCategory(Request $req, Response $res)
```

Comment:
@jjsimonAB if you already specify void to one function specify the same thing to all

Reply:
done